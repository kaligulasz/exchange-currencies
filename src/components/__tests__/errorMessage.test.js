import ErrorMessage from '../ErrorMessage';

describe('<ErrorMessage />', () => {
  it('renders children and message', () => {
    const componentWithChildren = shallow(
      <ErrorMessage message="example error message">
        <div>some children</div>
      </ErrorMessage>,
    );

    expect(componentWithChildren).toMatchSnapshot();
  });
});
