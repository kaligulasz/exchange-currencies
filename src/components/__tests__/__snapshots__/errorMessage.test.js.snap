// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Message /> renders children and message 1`] = `
<Message__MessageWrapper>
  <div>
    some children
  </div>
  example error message
</Message__MessageWrapper>
`;
